src/chunk.c

```diff
@@ -3411,44 +3334,59 @@ chunk_update_status(FormData_chunk *form)
 								   BTEqualStrategyNumber,
 								   F_INT4EQ,
 								   Int32GetDatum(chunk_id));
+	bool success = false;
+	bool dropped, dropped_isnull;
 
```

Comment:
Are you planning to make the changes to examine the result of the tuple lock in a different PR?
https://github.com/timescale/timescaledb/blob/8a40e553de46c7b6944bdfb45620b8a0ba2a4b6b/src/scanner.c#L414